src/main/java/lotto/domain/WinningNumber.java

```diff
@@ -1,20 +1,30 @@
 package lotto.domain;
 
-import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Stream;
 
 public class WinningNumber {
     private final List<Integer> winningNumbers;
+    private Integer bonusNumber;
 
-    public WinningNumber() {
-        winningNumbers = new ArrayList<>();
+    public WinningNumber(List<Integer> winningNumber) {
+        this(winningNumber, 0);
     }
 
-    public WinningNumber(List<Integer> winningNumber) {
-        this.winningNumbers = winningNumber;
+    public WinningNumber(List<Integer> winningNumbers, Integer bonusNumber) {
+        this.winningNumbers = winningNumbers;
+        this.bonusNumber = bonusNumber;
     }
 
     public List<Integer> getWinningNumbers() {
         return winningNumbers;
     }
+
+    public Stream<Integer> stream() {
+        return winningNumbers.stream();
+    }
+
+    public Integer getBonusNumber() {
+        return bonusNumber;
+    }
```

Comment:
getter말고 기능을 제공해보는건 어떨까요? 
[Tell don't ask](https://www.google.com/search?q=tell+don%27t+ask&oq=Tell+&aqs=chrome.1.69i57j69i59j69i61j69i60j69i61.1979j0j7&sourceid=chrome&ie=UTF-8)
```suggestion
    public boolean isContained(List<Integer> numbers) {
        return numbers.contains(bonusNumber);
    }
```